back-end/validations/checkSnacks.js

```diff
@@ -0,0 +1,44 @@
+/** @format */
+
+const checkName = (req, res, next) => {
+	const { name } = req.body;
+	if (name) {
+		next();
```

Comment:
I think we could add some more validation to this - min length maybe?  Use a regex to see if its not a string of numbers?